runtime-modules/storage/src/lib.rs

```diff
@@ -2978,6 +2958,25 @@ impl<T: Trait> Module<T> {
             Self::check_buckets_for_overflow(storage_buckets, &bag_change.voucher_update)?;
         }
 
+        // Validate distribution bucket IDs
```

Comment:
This has become a long function now with several sub validation routines. Can you please refactor each unit with self contained validation logic (like distribution bucket validation) into its own separate function. I think it is clearer that way

Reply:
Fixed here: https://github.com/Joystream/joystream/pull/3348/commits/32d252427ebd6c3b417662badd523e572ab3cbc4